### PR TITLE
Bug#1501597 - Updated the note to higlight not to use usernames

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://bugzilla.redhat.com/show_bug.cgi?id=1501597

### CODE REVIEW

File: dev_guide/builds/build_inputs.adoc [hunk 1]
```diff
@@ -333,14 +333,23 @@ A URI pattern must consist of:
...
-https://github.com/openshift/origin.git. They do not match the alternate SSH
-style that Git also uses. For example, git@github.com:openshift/origin.git.
+`https://github.com/openshift/origin.git`. They do not match the alternate SSH
+style that Git also uses. For example, `git@github.com:openshift/origin.git`.
 
 It is not valid to attempt to express a URI pattern in the alternate style, or
 to include a username/password component in a URI pattern.
+
+For example, if you use `ssh://git@bitbucket.atlassian.com:7999/ATLASSIAN/jira.git` for a git repository URL, the source secret must be specified as `pass:c[ssh://bitbucket.atlassian.com:7999/*]` (and not `pass:c[ssh://git@bitbucket.atlassian.com:7999/*]`).
+
```
Comment (gaurav-nelson) [inline dev_guide/builds/build_inputs.adoc]: There were formatting errors without `pass:c[ ]`. It was treating the content between *'s as bold.

**Comment by gaurav-nelson** [comment]
@tanaka-takayoshi PTAL 🍅

**Comment by tanaka-takayoshi** [comment]
@gaurav-nelson It looks good to me.

**Comment by wzheng1** [comment]
LGTM, thx

**Comment by gaurav-nelson** [comment]
@bfallonf PTAL

File: dev_guide/builds/build_inputs.adoc [hunk 2]
```diff
@@ -333,14 +333,23 @@ A URI pattern must consist of:
 
 In all of the above, a `*` character is interpreted as a wildcard.
 
-[NOTE]
+[IMPORTANT]
 ====
 URI patterns only match Git source URIs which are conformant to RFC3986. For example,
```
Comment (bfallonf) [inline dev_guide/builds/build_inputs.adoc]: @gaurav-nelson should we link to the RFC?

File: dev_guide/builds/build_inputs.adoc [hunk 3]
```diff
@@ -333,14 +333,23 @@ A URI pattern must consist of:
...
-https://github.com/openshift/origin.git. They do not match the alternate SSH
-style that Git also uses. For example, git@github.com:openshift/origin.git.
+`https://github.com/openshift/origin.git`. They do not match the alternate SSH
+style that Git also uses. For example, `git@github.com:openshift/origin.git`.
 
 It is not valid to attempt to express a URI pattern in the alternate style, or
 to include a username/password component in a URI pattern.
```
Comment (bfallonf) [inline dev_guide/builds/build_inputs.adoc]: The above is expletive. Maybe something like "Expressing a URI pattern in an alternate style, or including a username/password component in a URI pattern will result in an invalid pattern".

**Comment by bfallonf** [comment]
@gaurav-nelson What you've done looks good, but I've made some comment on the couple sentences around it. Just suggestions. Otherwise LGTM :christmas_tree:

**Comment by gaurav-nelson** [comment]
Thank you @bfallonf Can you please check this again, I have simplified it.

**Comment by bfallonf** [comment]
@gaurav-nelson LGTM :+1:

**Pull request merged**